### PR TITLE
Prepare the 0.10.5 release

### PR DESCRIPTION
**Pull request by benjyw**

### CODE REVIEW

**Review by kaos — APPROVED** [review]

**Comment by jsirois** [comment]
I suspect this runs afoul of https://github.com/pantsbuild/scie-pants/pull/319#issuecomment-1793564069

Although I submitted a patch to dotenvs here: https://github.com/arniu/dotenvs-rs/pull/5 there has been no activity. I feel good enough about the rules posted by that crate (https://github.com/arniu/dotenvs-rs#parsing-rules) to advertise those rules for scie-jump .env support and depend on my fork. I'll try to get out a scie-jump release today that switches from dotenvy to my fork of dotenvs making this sort of upgrade safer.

**Comment by kaos** [comment]
Thanks for jumping in John. I was surprised this wasn't upgraded as I had a vague recollection of seeing a PR about upgrading scie-jump not too long ago. Didn't connect to look for the PR though, my bad.

**Comment by jsirois** [comment]
https://github.com/a-scie/jump/releases/tag/v0.14.0 is now released should allow guilt-free upgrading. It does not address @huonw's request that `X=a $Y` produce an error, since a shell usually would (if $Y expanded to a non-command). But that request seems a bridge too far to me / has scie-jump nanny on behalf of shell for shell when the .env file may never be sourced anyhow and, even if source, may even work as intended.

**Comment by benjyw** [comment]
Bumped to scie-jump 0.14.0, so should be good to go now.

File: CHANGES.md
Comment (kaos) [inline CHANGES.md:1]: Could perhaps also mention the bug fixes addressed regarding `.env` file handling?

**Review by kaos — APPROVED** [review]

**Comment by huonw** [comment]
What's the status of this one @benjyw?

**Comment by benjyw** [comment]
> What's the status of this one @benjyw?

Oops, fell off my radar. I've updated the CHANGES.md per @kaos's suggestion. Will merge when green.

**Pull request merged**